### ISSUE TITLE
Custom Validation Attributes for Array Type Form Fields

### ISSUE THREAD

**Issue by HarleyFuagras** [proengsoft/laravel-jsvalidation#193]
Hi,
Having an array type form field like...

```
<div class="form-group">
    {!! Form::label("es[title]", trans("messages.title"), ["class" => "control-label"]) !!}
    {!! Form::text("es[title]", getFormInput($entry, "title", $locale), ["class" => "form-control", "id" => "es[title]"]) !!}
</div>
```

Validated with a CreateRequest as follows...

```class PostCreateRequest extends Request
{
    /**
     * Get the validation rules that apply to the request.
     *
     * @return array
     */
    public function rules()
    {
        return [
             'es.title' => 'required|max:255',           
        ];
    }
}
```

I don't know how to change the attribute placeholders with more friendly texts at resources/lang/validation.php file. I have tried the following options but any of them is working. The form is being validated naming the field 'es.title' but the error message showed is not replacing the field name correctly even if I name the attributes array key also 'es.title'. Any ideas?

```
'attributes' => [      
        'title' => 'título',
        //'es.title' => 'título',       
       //'es[title]' => 'título',       
    ],
```

**Comment by aleixfabra** [comment]
👍

**Comment by torrentalle** [comment]
This issue is fixed in `v1.4.2`
